landing-page/src/index.css

```diff
@@ -116,7 +111,7 @@ code {
   border-style: solid;
 }
 
-.button:hover{
+.button:hover, .secondaryButton:hover{
```

Comment:
Seems like you don't need  `secondaryButton` class as it doesn't have any unique styles